solana/program_v2_cli/.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
delete this folder `.circleci` (circle is an alternative to github actions)

Reply:
CC @DiosAkhilleus

Reply:
done